### ISSUE TITLE
App crashes after opening susi skills

### ISSUE THREAD

**Issue by Bhavnaharitsa** [fossasia/susi_android#1898]
**Describe the bug**

The app crashes as soon as opening susi-skills

**Expected Behaviour**

The app should open skills

**Screenshots of the issue**

![screenshot_20190201-220453_samsung experience home](https://user-images.githubusercontent.com/37215508/52136497-380aca00-266e-11e9-9934-af101d039421.jpg)

**Would you like to work on the issue?**

Yes!

**Comment by aleronarjun** [comment]
Hello @Bhavnaharitsa can I work on this issue?
Thanks.

**Comment by andy5995** [comment]
I experience this too. But the message that displays for me is "the package installer has stopped". I just installed susi from the fdroid repo. My phone is a Samsung galaxy j7 running android 8.1.0.

Version 1.0.13

Didn't happen as soon as opening skills, but after I browsed skills for a minute and hit the back button. 

I can close the error box and the app, but every time I try to reopen, I get the error message instead.
